ui/common/build.gradle.kts

```diff
@@ -14,6 +17,22 @@ kotlin {
                 implementation(projects.data.network)
                 api(libs.com.arkivanov.decompose)
                 api(libs.com.arkivanov.decompose.extensions.compose.jetbrains)
+                implementation("com.alialbaali.kamel:kamel-image:0.4.0")
+                implementation("it.skrape:skrapeit:1.2.2")
+
+            }
+        }
+        val commonTest by getting {
+            dependencies {
+                implementation(kotlin("test"))
+                implementation(libs.org.jetbrains.kotlin.test.common)
+                implementation(libs.org.jetbrains.kotlin.test.annotations.common)
```

Comment:
Formatting here looks off, but it could be because I'm on my phone

Reply:
ran spotless apply just in case